flasher_stub/Makefile

```diff
@@ -55,7 +55,7 @@ SRCS = stub_flasher.c slip.c stub_commands.c stub_write_flash.c stub_io.c
 SRCS_8266 = miniz.c
 
 BUILD_DIR = build
-ESPTOOL_STUBS_DIR = ..
+ESPTOOL_STUBS_DIR = $(ESPTOOL_PATH)/esptool/targets/stub_flasher/1
```

Comment:
This looks like a temporary change. Please revert it.

Reply:
`ESPTOOL_STUBS_DIR` is used with the `make install` cmd to replace the just-build JSON stub files in esptool directory for immediate usage in the context of esptool. With `ESPTOOL_STUBS_DIR = ..`, this is not possible. This was therefore left in intentionally.

Reply:
I'm sorry but I still don't like to hardcode such path in this project which introduces another environment variable not defined or documented, and pointing to a project which includes this one (circular dependency).